### DISCUSSION TITLE
Is there any progress on the technology stack for the Open Compute project ?

### DISCUSSION THREAD

**Discussion by BhujayKumarBhatta** [PeoplePlusAI/Open-Cloud-Compute-OCC#14] [Q&A] [answered]
Can we build this on OpenStack ( Currently Named as Open Infra ) as the foundation for the Cloud Orchestration.

**Comment by tanvipeopleplusai** [comment, marked as answer]
Hi @BhujayKumarBhatta, we have been working on the tech stack for this. We will be releasing a technical roadmap in early May. I'll drop the link here once it is published.

**Reply by aarvee11** [reply]
Openstack can be a provider for sure as it offers infrastructure on API by design
